src/schema/posts.ts

```diff
@@ -1550,6 +1667,96 @@ export const resolvers: IResolvers<any, Context> = {
       });
       return { _: true };
     },
+    votePost: async (
```

Comment:
Just checking should we not also support the "old" data here for the time being? (not sure, just raising it as a question for now)

Reply:
After migration PR there is no point in writing to old tables since all data will also be copied into `user_post`. Old migrations are there only to support BC for extension. So idea is to keep these new ones clean and when we start using them their data will already be filled.